chirp/wxui/query_sources.py

```diff
@@ -515,7 +515,7 @@ def build(self):
         return vbox
 
     def populateprov(self, event):
-        self.selected_province(event.GetString)
+        self.selected_province(provchoice.GetStringSelection())
```

Comment:
Hmm, provchoice isn't in scope here is it? Ohh, it's global. Hmm, that's why I changed it, thinking that moving it out of the build function would lose sight of it. That's probably not the best plan, to have UI elements be global, especially if they could get re-used or re-parented by a second invocation. Better to store them on self so they live and die with the dialog itself. Did you do that because you need to access them from the other event handlers, or to try to avoid re-building them at some point? Storing them on self makes them accessible later and avoids them needing to be global.

Also, I think `event.GetString()` will make this work (if properly called as a function).